ansible_collections/arista/avd/docs/how-to/first-cvp-avd-project.md

```diff
@@ -2,28 +2,28 @@
 
 This document explains how to customize demo information and set up an environment. A complete Git repository is available and can be used as support for this how-to document.
 
-__Demo repository__: [arista-netdevops-community/ansible-avd-cloudvision-demo](https://github.com/arista-netdevops-community/ansible-avd-cloudvision-demo)
+**Demo repository**: [arista-netdevops-community/ansible-avd-cloudvision-demo](https://github.com/arista-netdevops-community/ansible-avd-cloudvision-demo)
 
 ## Installation Process
 
 ### Demonstration requirements
 
 #### Components
 
-- A __CloudVision Portal (CVP)__ instance running a minimum version `2018.2.5` or __CloudVision as a Service (CVaaS)__.
+- A **CloudVision Portal (CVP)** instance running a minimum version `2018.2.5` or **CloudVision as a Service (CVaaS)**.
 - A network topology:
   - Demo is based on a two spines / four leafs running on GNS3 but could be configured on EVE-NG platform.
-  - Any physical or virtual topology with __out of band (OOB)__ connected to CVP should work.
+  - Any physical or virtual topology with **out of band (OOB)** connected to CVP should work.
 - A Python environment with CloudVision access.
 
 ![Lab Topology](../_media/first-cvp-avd-topology.png)
 
 #### IP Address management
 
-- ___CloudVision IP address___:
+- _**CloudVision IP address**_:
```

Comment:
Since we go for "star" style everywhere, I guess this should be
```suggestion
- ***CloudVision IP address***:
```

Same elsewhere

Reply:
I will fix these, I'll run some tests as well. Fairly odd that it corrected to that format

Reply:
Fixed these issues, Seems like triple underscore causes a rewrite to _** vs ***, if using *** from the start, then there is no rewrite

Reply:
Julio fixed and added a new rule to ensure consistency MD049